modules/cdc-ext/src/main/java/org/apache/ignite/cdc/metrics/KafkaToIgniteMetrics.java

```diff
@@ -0,0 +1,214 @@
+    private AtomicLongMetric markersCnt;
+
+    /** Standalone kernal context. */
+    private StandaloneGridKernalContext kctx;
+
+    /** CDC metrics registry. */
+    private MetricRegistryImpl mreg;
+
+    /** */
+    private final IgniteLogger log;
+
+    /** Streamer configuration. */
+    private final KafkaToIgniteCdcStreamerConfiguration streamerCfg;
+
+    /** Ignite instance name. */
+    private final String igniteInstanceName;
+
+    /** */
+    private KafkaToIgniteMetrics(
+        IgniteLogger log,
+        KafkaToIgniteCdcStreamerConfiguration streamerCfg,
+        String igniteInstanceName
+    ) throws IgniteCheckedException {
+        this.log = log;
+        this.streamerCfg = streamerCfg;
+        this.igniteInstanceName = igniteInstanceName;
+
+        initStandaloneMetricsKernal();
+        initMetrics();
+    }
+
+    /**
+     * Creates an instance of {@link KafkaToIgniteMetrics}.
+     * @param log Logger.
+     * @param streamerCfg Streamer config.
+     * @param igniteInstanceName Ignite instance name.
+     * @return {@link KafkaToIgniteMetrics} instance.
+     */
+    public static KafkaToIgniteMetrics startMetrics(
+        IgniteLogger log,
+        KafkaToIgniteCdcStreamerConfiguration streamerCfg,
+        String igniteInstanceName
+    ) {
+        try {
+            return new KafkaToIgniteMetrics(log, streamerCfg, igniteInstanceName);
+        }
+        catch (IgniteCheckedException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /** @throws IgniteCheckedException If failed. */
+    private void initStandaloneMetricsKernal() throws IgniteCheckedException {
+        kctx = new StandaloneGridKernalContext(log, null, null) {
+            @Override protected IgniteConfiguration prepareIgniteConfiguration() {
+                IgniteConfiguration cfg = super.prepareIgniteConfiguration();
+
+                cfg.setIgniteInstanceName("kafka-ignite-streamer-" + igniteInstanceName);
+
+                if (!F.isEmpty(streamerCfg.getMetricExporterSpi()))
+                    cfg.setMetricExporterSpi(streamerCfg.getMetricExporterSpi());
+                else {
+                    cfg.setMetricExporterSpi(U.IGNITE_MBEANS_DISABLED
+                        ? new NoopMetricExporterSpi()
+                        : new JmxMetricExporterSpi());
+                }
+
+                initializeDefaultMBeanServer(cfg);
+
+                return cfg;
+            }
+
+            /** {@inheritDoc} */
+            @Override public String igniteInstanceName() {
+                return config().getIgniteInstanceName();
+            }
+        };
+
+        startAllComponents(kctx);
+
+        for (IgniteSpi metricSpi : kctx.config().getMetricExporterSpi())
+            metricSpi.onContextInitialized(new StandaloneSpiContext());
+    }
+
+    /** Initialize metrics. */
+    private void initMetrics() {
+        mreg = kctx.metric().registry(metricName("kafka-to-ignite-metrics"));
```

Comment:
We have metric group 'cdc', and 'cdc.consumer' for all types of `IgniteTo...` consumers. Perhaps something like 'cdc.applier' will be more relevant here.

Reply:
Refactored the naming for CDC metrics registries